src/standard/expr.ml

```diff
@@ -1569,6 +1649,17 @@ module Term = struct
       mk' ~builtin:Builtin.Coercion "coerce"
         [a; b] [Ty.of_var a] (Ty.of_var b)
 
+    let in_interval (b1, b2) = mk'
+        ~name:"in_interval" ~builtin:(Builtin.In_interval (b1, b2))
+        "InInterval" [] [Ty.int; Ty.int; Ty.int] Ty.prop
+
+    let maps_to =
+      let a = Ty.Var.mk "alpha" in
+      let b = Ty.Var.mk "beta" in
+      mk'
+        ~name:"maps_to" ~builtin:Builtin.Maps_to
+        "MapsTo" [] [Ty.of_var a; Ty.of_var b] Ty.prop
```

Comment:
This is wrong: the type of `maps_to` is not closed. You should add the `a` and `b` type variables in the first list, so that they are properly bound in the type of the `maps_to` constant.

Reply:
Yes. Done.